Sources/AsyncStream.swift

```diff
@@ -1,18 +1,24 @@
 public protocol AsyncSending {
-    func send(_ data: Data, timingOut deadline: Double, completion: (Void throws -> Void) -> Void)
+    associatedtype Sent
+    func send(_ data: Sent, timingOut deadline: Double, completion: (Void throws -> Void) -> Void)
```

Comment:
Yes! I've brought this idea up quite a few times but it doesn't look like anyone else agrees.